src/tests/JIT/Regression/JitBlue/Runtime_100466/Runtime_100466.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Xunit;
+
+public static class Runtime_100466
```

Comment:
Nit: I realize this is annoying, but I think this sort of test should possibly be under `runtime/src/tests/Loader`. It can also be named something like `StaticTypeInitialization`, which would be clearer. I'm also fine with it not being renamed, but the location under `Loader` does seem more appropriate.

Reply:
> `StaticTypeInitialization`

Perhaps `StaticValueTypeInitialization` to be more specific?

Reply:
I do not think about this issue as a classloader bug.

This bug was in the implementation of boxing. Boxing implementation had a superfluous cctor trigger on one path that made it do more than what it was supposed to in some cases. Boxing is a JIT-centric concept - the basic tests are under https://github.com/dotnet/runtime/tree/main/src/tests/JIT/Methodical/Boxing.

The bug intersects with cctor triggers. Cctor triggers are JIT-centric concept too - the basic tests are under https://github.com/dotnet/runtime/tree/main/src/tests/JIT/Methodical/cctor.

The bug is not in the type initialization management, like the logic that caches exceptions thrown by cctors or that breaks cycles between cctors. Type initialization management would be classloader - the basic tests are under https://github.com/dotnet/runtime/tree/main/src/tests/Loader/classloader/TypeInitialization.

The intersection of JIT/Methodical/Boxing and JIT/Methodical/cctor gives JIT/Regressions.

This is my though process for the current location, but I do not have a strong opinion about it. I am happy to move it if you tell me where you would like to see it moved to exactly.

Reply:
Okay, that makes sense.